### PR TITLE
Fix GIF not display with EOF error

### PR DESCRIPTION
**Pull request by s1rius**

### CODE REVIEW

**Comment by s1rius** [comment]
Fix #533 
When I load a GIF image with EOF error. It can display on screen.
But the last frame display alpha pixel error. Could some one help me?
d

**Comment by facebook-github-bot-0** [comment]
@s1rius updated the pull request.

File: animated-gif/src/main/jni/gifimage/gif.cpp [hunk 1]
```diff
@@ -421,9 +425,10 @@ int modifiedDGifSlurp(GifWrapper* pGifWrapper) {
 
   pGifFile->ExtensionBlocks = NULL;
   pGifFile->ExtensionBlockCount = 0;
+  bool isStop = false;
```
Comment (tyronen) [inline animated-gif/src/main/jni/gifimage/gif.cpp]: call this exitLoop

**Comment by tyronen** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/485270938337348/int_phab) to review.

**Comment by facebook-github-bot-4** [comment]
@s1rius updated the pull request.

**Comment by tyronen** [comment]
Hi, I tried patching this, and discovered that if I ran our comparison app, GIFs would no longer load with this PR. Were you able to get it to work?

**Comment by s1rius** [comment]
Please add `compile project('animated-gif')` on comparison app.

**Comment by s1rius** [comment]
Please add `compile project('animated-gif')` on dependencies.

**Comment by s1rius** [comment]
It works after this [commit](https://github.com/facebook/fresco/commit/63532215f9b22b36d7d369af251dffd58150fca5).
And all GIFs mentioned in #167 issue display with this PR.
Please review. @tyronen

**Comment by facebook-github-bot-0** [comment]
@s1rius updated the pull request.

**Comment by tyronen** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/485270938337348/int_phab) to review.

**Pull request closed without merging**